Sources/WebSocketCompression/PermessageDeflate.swift

```diff
@@ -90,6 +90,10 @@ public class PermessageDeflateCompressor: Deflater {
 
         return outputBuffer
     }
+
+    public func end() {
+        deflateEnd(&stream)
```

Comment:
Hi Pushkar, Since no other class is accessing `stream` outside deflater/inflater can it be made `internal` instead of `public` now?